AppCenterData/AppCenterDataTests/MSDataOperationProxyTests.m

```diff
@@ -29,7 +31,8 @@ - (void)setUp {
   [super setUp];
 
   // Init properties.
-  _documentStoreMock = OCMClassMock([MSDBDocumentStore class]);
+  MSDBStorage *dbStorage = [[MSDBStorage alloc] initWithVersion:0 filename:kMSDBDocumentFileName];
```

Comment:
Why do we need this? I don't see any change on tests.

Reply:
There is a function which does update on the local store and does some validation on expiration, pending operation and then actual upsert on the document. It used to reside in the `MSDataOperationProxy` class but moved it to the `MSDBDocumentStore` layer cos list needs it extensively. So the tests which depend on this validation/upsert function needs a partial mock of the documentStore.

Reply:
The function is called `updateDocumentWithToken` in the class `MSDBDocumentStore`